config/operator/olm.cue

```diff
@@ -174,21 +174,14 @@ _related_images: [
 	}
 ]
 
-if parameters.hubbleUIProxyImage != "nothing" {
-	_related_images: _related_images + [{
-		name: "hubble-ui-proxy"
-		image: parameters.hubbleUIProxyImage
-	}]
-}
-
-// quay.io/coreos/etcd is not included in Cilium >= 1.15 Helm charts.
-// Ref: https://github.com/cilium/cilium/blob/v1.15.1/install/kubernetes/cilium/values.yaml
-if strings.HasPrefix(parameters.ciliumVersion, "1.12") || strings.HasPrefix(parameters.ciliumVersion, "1.13") || strings.HasPrefix(parameters.ciliumVersion, "1.14") {
-	_related_images: _related_images + [{
```

Comment:
this didn't work, i guess cue doesn't allow mutating lists, i got this error: `cue: marshal error: template.items.14.spec.relatedImages: cycle detected`